provider/gce/environ_broker_test.go

```diff
@@ -233,7 +234,7 @@ func (s *environBrokerSuite) TestGetDisks(c *gc.C) {
 		}
 	}
 
-	diskSpecs, err := gce.GetDisks(s.spec, s.StartInstArgs.Constraints, "trusty", "32f7d570-5bac-4b72-b169-250c24a94b2b", true)
+	diskSpecs, err := gce.GetDisks(s.spec, s.StartInstArgs.Constraints, "trusty", "32f7d570-5bac-4b72-b169-250c24a94b2b", gce.UbuntuDailyImageBasePath)
```

Comment:
This now doesn't do anything - you need a test where the model config has stream set to "daily" and then check the expected image path

Reply:
Have added some tests. Could you please cast your eye over them and I'll fire off the merge?